### PR TITLE
Space preview component

### PR DESCRIPTION
**Pull request by deekshithepns**
send this kind of link to test preview... http://localhost:3000/chat/spaceid:3e1b3c01ad76a79e998d535111e935bd1bb7d2f5f4cb9ec62ee359e81fdf90b9

### CODE REVIEW

File: src/components/chat/w2wChat/chats/Chats.tsx
Comment (abhishek-01k) [inline src/components/chat/w2wChat/chats/Chats.tsx:1]: Screenshot 2023-04-26 at 12 09 39 PM

Getting this undefined image and description when sending the url in the message link

File: src/components/space/spaceReusables/RequestBody.tsx
Comment (abhishek-01k) [inline src/components/space/spaceReusables/RequestBody.tsx:1]: Screenshot 2023-04-26 at 11 45 48 AM

here the time is not showing properly.

Comment (abhishek-01k) [reply src/components/space/spaceReusables/RequestBody.tsx:1]: Similarly time is not showing properly on the sideBox in Requests section

**Review by abhishek-01k — COMMENTED** [review]
Screenshot 2023-04-26 at 12 13 03 PM

Here on the spaceBox padding should be there according to the designs.

**Review by abhishek-01k — APPROVED** [review]

File: src/components/chat/w2wChat/chats/spacePreviewCards/SpacePreviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
+// React + Web3 Essentials
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/chats/spacePreviewCards/SpacePreviewCard.tsx:1]: this file is not used anywhere, can be deleted it

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

**Review by mishramonalisha76 — APPROVED** [review]

**Pull request merged**